src/MMLib.SwaggerForOcelot/Repositories/DownstreamSwaggerDocsRepository.cs

```diff
@@ -42,7 +42,8 @@ public class DownstreamSwaggerDocsRepository : IDownstreamSwaggerDocsRepository
             string docsVersion = null)
         {
             string url = await GetUrlAsync(route, endPoint, docsVersion);
-            HttpClient httpClient = _httpClientFactory.CreateClient();
+            var clientName = (route?.DangerousAcceptAnyServerCertificateValidator ?? false) ? "HttpClientWithSSLUntrusted" : string.Empty;
```

Comment:
Please extract this name to a constant and use it in both places.

Reply:
Done